lib/IDE/SyntaxModel.cpp

```diff
@@ -418,6 +418,21 @@ CharSourceRange parameterNameRangeOfCallArg(const TupleExpr *TE,
   return CharSourceRange();
 }
 
+static CharSourceRange findDocCommentRange(DeclAttributes Attrs) {
+  for (auto Attr : Attrs) {
+    if (auto DocAttr = dyn_cast_or_null<RawDocCommentAttr>(Attr)) {
+      return DocAttr->getCommentRange();
```

Comment:
`RawDocCommentAttr::getCommentRange` returns *raw* comment range. For example:
```swift
// Copyright (c) FooBar Inc.

/// secret.
func doSomething() {
```
IIRC, `getCommentRange()` returns from `// Copy` on line 1 to `secret.\n` on line 3.
Is that desired behavior?